mentoring/choosing_a_solution.md

```diff
@@ -70,6 +69,7 @@ This can help your explanations be simpler and clearer.
 The third tab is called "Guidance".
 If you click on this you'll see some information that might be useful for you:
 
+- **The Exemplar solution** Try to guide the student towards this solution. It is the best place for them to reach at this point during the Track. You may discover that your approach differs significantly from the exemplar solution. This may be due to you being familiar with more advanced techniques than the student. Keep in mind that you can only expect the student to be familiar with the concepts they've learnt while completing the learning path to the exercise they are working on. Consider this fact when providing your feedback and try not to overwhelm your student with knowledge they may not be prepared for yet.
```

Comment:
```suggestion
- **The Exemplar solution** Try to guide the student towards this solution.
  It is the best place for them to reach at this point during the Track.
  You may discover that your approach differs significantly from the exemplar solution.
  This may be due to you being familiar with more advanced techniques than the student.
  Keep in mind that you can only expect the student to be familiar with the concepts they have learned while completing the learning path to the exercise they are working on.
   Consider this fact when providing your feedback and try not to overwhelm your student with knowledge they may not be prepared for yet.
```